src/OpenTelemetry/Metrics/MeterProviderSdk.cs

```diff
@@ -13,16 +13,19 @@ namespace OpenTelemetry.Metrics;
 
 internal sealed class MeterProviderSdk : MeterProvider
 {
+    internal const string EmitOverFlowAttributeConfigKey = "OTEL_DOTNET_EXPERIMENTAL_METRICS_EMIT_OVERFLOW_ATTRIBUTE";
+    internal const string ReclaimUnusedMetricPointsConfigKey = "OTEL_DOTNET_EXPERIMENTAL_METRICS_RECLAIM_UNUSED_METRIC_POINTS";
+    internal const string ExemplarFilterConfigKey = "OTEL_DOTNET_EXPERIMENTAL_METRICS_EXEMPLAR_FILTER";
```

Comment:
mm.. The official name of the env variable is "OTEL_METRICS_EXEMPLAR_FILTER", though that is still experimental.
https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/configuration/sdk-environment-variables.md#exemplar

Should we use the official env variable, and support it only in pre-release builds? 
(I understand that other env vars have "Experimental" in it, but that is covering the SDK's own experimental feature.)